samples/Diagnostics.StatusCodes.Mvc/Controllers/HomeController.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.AspNet.Mvc;
+
+namespace Diagnostics.StatusCodes.Mvc.Controllers
+{
+    public class HomeController : Controller
+    {
+        // GET: /<controller>/
+        public string Index()
+        {
+            return "Hello World, try /bob";
```

Comment:
Does this work because `/bob` causes a 404 or 500 or something and thus an error is shown? Might want to clarify that.